docs/topics/logging.txt

```diff
@@ -466,7 +466,8 @@ this logger.
 
 Log messages related to the handling of requests. 5XX responses are
 raised as ``ERROR`` messages; 4XX responses are raised as ``WARNING``
-messages.
+messages. Messages logged to ``django.security`` are not logged to
+``django.request``.
```

Comment:
I found this sentence a little confusing. On the face of it, it seems redundant to say that messages that log to `django.security` don't get logged to `django.request`. I think you mean that *requests* that cause a message to `django.security` don't cause a message to log to `django.request`.

Maybe reword this as something like "Requests with errors that trigger a log to the `django.security` logger are not logged to `django.request`".

Note that there's some text below in the `django.security` section that says much the same thing:

> "Requests resulting in a 400 response due to a ``SuspiciousOperation`` will not be logged to the ``django.request`` logger, but only to the ``django.security`` logger."

I don't think all that detail is appropriate here, but maybe link to that paragraph so that users can see more information if they want to?

Also, this paragraph could re rewrapped at 79 characters.